migrations/20200713233601-add-hospital-column-to-patients.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.sequelize.transaction(t => {
+      return Promise.all([
+        queryInterface.addColumn(
+          "patients",
+          "hospitalId",
+          {
+            type: Sequelize.DataTypes.INTEGER
+          },
+          { transaction: t }
+        )
+      ]);
+    });
+  },
+
+  down: (queryInterface, Sequelize) => {
+    return queryInterface.sequelize.transaction(t => {
+      return Promise.all([
+        queryInterface.removeColumn(
+          "patients",
+          "hospitalId",
+          {
+            type: Sequelize.DataTypes.INTEGER
+          },
+          { transaction: t }
+        )
+      ]);
+    });
```

Comment:
same thing here except the type is unnecessary

```suggestion
    return queryInterface.removeColumn("patients", "hospitalId");
```